bridge-proxy/src/config.rs

```diff
@@ -7,6 +7,10 @@ pub trait ConfigModule {
     #[storage_mapper("pendingTransactions")]
     fn pending_transactions(&self) -> MapMapper<usize, EthTransaction<Self::Api>>;
 
+    #[view(refundTransactions)]
```

Comment:
ok, this view can be used to "select" only the transaction you want to refund in case we do not put a "general" sc calls executors for refunds